src/components/application_manager/src/application_impl.cc

```diff
@@ -347,6 +355,40 @@ void ApplicationImpl::set_is_media_application(bool is_media) {
   is_media_ = is_media;
 }
 
+void ApplicationImpl::set_audio_streaming_indicator(
+    mobile_apis::AudioStreamingIndicator::eType indicator) {
+  audio_streaming_indicator_ = indicator;
+}
+
+bool ApplicationImpl::AddIndicatorWaitForResponse(
+    mobile_api::AudioStreamingIndicator::eType indicator) {
+  if (indicator == audio_streaming_indicator()) {
```

Comment:
@dtrunov this is great example why const is good. In case if you ( or someone else_ will make typo ```if(indicator = audio_streaming_indicator())``` const variable will return error. 
But in that case please prefer but variables on the right side of comparation any way